announcements/_posts/2019-11-22-government.md

```diff
@@ -0,0 +1,35 @@
+---
+author: Tim Zabel (@Tjzabel)
+title: "GitLab in Government"
```

Comment:
Mind if we make this `Open Source in the Federal Government`

Reply:
sure thing